clearwater-cassandra/usr/share/clearwater/infrastructure/scripts/cassandra_schemas/run_cassandra_schemas

```diff
@@ -12,6 +12,9 @@ for SCHEMA in $(ls -1 /usr/share/clearwater/cassandra-schemas/* 2>/dev/null)
 do
   if [ -f "$SCHEMA" ]; then
     $SCHEMA
+    rc=$?
+    if [ $rc != 0 ]; then
```

Comment:
There are potentially multiple schema scripts - returning without looking at the return code would mean that this would only run one schema script.

This code changes the script so that it exits as soon as any one schema script fails.